Move file py.typed to folder exasol/bucketfs

See #173 

Currently, the file `py.typed` is in the root folder of bucketfs.

Which led to the following [error message in SLC-tool](https://github.com/exasol/script-languages-container-tool/actions/runs/12988482820/job/36219628003?pr=260#step:4:14) for example: 

```
exasol/slc/internal/tasks/test/upload_file_to_bucket_fs.py:5:1: error: Skipping
analyzing "exasol.bucketfs": module is installed, but missing library stubs or
py.typed marker  [import-untyped]
    import exasol.bucketfs as bfs
    ^
```

--- a/doc/changes/changelog.md
+++ b/doc/changes/changelog.md
@@ -1,6 +1,7 @@
 # 📝 Changes
 
 * [unreleased](unreleased.md)
+* [1.0.1](changes_1.0.1.md)
 * [1.0.0](changes_1.0.0.md)
 * [0.14.0](changes_0.14.0.md)
 * [0.13.0](changes_0.13.0.md)
@@ -22,6 +23,7 @@
 hidden:
 ---
 unreleased
+changes_1.0.1
 changes_1.0.0
 changes_0.14.0
 changes_0.13.0

--- a/doc/changes/changes_1.0.1.md
+++ b/doc/changes/changes_1.0.1.md
@@ -1,0 +1,5 @@
+# 1.0.1 - 2025-01-27
+
+## Refactorings
+
+* #191: Moved file py.typed to folder exasol/bucketfs

--- a/doc/changes/unreleased.md
+++ b/doc/changes/unreleased.md
@@ -1,5 +1,1 @@
 # Unreleased
-
-## Refactorings
-
-* #191: Moved file py.typed to folder exasol/bucketfs

--- a/doc/changes/unreleased.md
+++ b/doc/changes/unreleased.md
@@ -1,1 +1,5 @@
 # Unreleased
+
+## Refactorings
+
+* #191: Moved file py.typed to folder exasol/bucketfs

--- a/exasol/bucketfs/version.py
+++ b/exasol/bucketfs/version.py
@@ -6,5 +6,5 @@
 # If you need to change the version, do so in the project.toml, e.g. by using `poetry version X.Y.Z`.
 MAJOR = 1
 MINOR = 0
-PATCH = 0
+PATCH = 1
 VERSION = f"{MAJOR}.{MINOR}.{PATCH}"

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -9,7 +9,7 @@ name = "exasol-bucketfs"
 packages  = [
     {include = "exasol"},
 ]
-version = "1.0.0"
+version = "1.0.1"
 description = "BucketFS utilities for the Python programming language"
 
 license = "MIT"

